backend/app/config_download/config_download_service.py

```diff
@@ -70,16 +75,15 @@ def _parse_neighbors(self, neighbors_string: str, origin_name: str) -> list[Conn
             ))
         return connections
 
-    def _get_device_type(self, device: NetmikoDevice):
-        switch_regex = r"^S\d{2}"
-        router_regex = r"^R\d{2}"
+    def _get_device_type(self, device_type_str: str):
+        device_type_str = device_type_str.lower()
```

Comment:
Do klasy DeviceType
```
@staticmethod
def from_str(s: str):
    match s.lower():
        case "dupa":
            return DeviceType.DUPA
```

Reply:
Tu nie mogę zrobić takiego prostego matcha, bo to nie jest taki trywialny string. Jedyne co sensownego znalazłem nie korzystające z heurystyk, że np. "switch to ma dużo więcej portów niż router" to komenda `show version`, która daje dość pokaźny output z jakimiś informacjami o systemie itp. w tym pojawia się linijka w stylu `Router uptime is 27 minutes, 50 seconds` (dla switcha wiadomo co jest zmienione)

Ale ze zrobieniem statycznej metody się zgadzam